pom.xml

```diff
@@ -324,6 +324,16 @@
 			<artifactId>jcip-annotations</artifactId>
 			<version>1.0</version>
 		</dependency>
+                <dependency>
```

Comment:
Can you change the scope of this to test please?

Reply:
Whoops, sorry: https://github.com/jscep/jscep/pull/89/commits/5270d4888b6d1001d25b367033b155bd368dfc86